syntax = "proto3";
import "google/protobuf/empty.proto";

message GetRequest {
  string key = 1;
}

message GetResponse {
  string key = 1;
  string value = 2;
}

message PutRequest {
  string key = 1;
  string value = 2;
}

message DeleteRequest {
  string key = 1;
  string value = 2;
}

service VolumeServer {
  rpc Get (GetRequest) returns (GetResponse) {}
  rpc Put (PutRequest) returns (google.protobuf.Empty) {}
  rpc Delete (DeleteRequest) returns (google.protobuf.Empty) {}
}
